src/components/organisms/pages/page_school/School.jsx

```diff
@@ -0,0 +1,46 @@
+// import { Button, Icon } from "/src/components/Organisms/pages/index.js";
```

Comment:
Does this comment have any relevance or usefulness?

Reply:
I forgot to use the index barrel instead of the individual imports, now it's corrected